### ISSUE TITLE
Ckeditor - customConfig not working

### ISSUE THREAD

**Issue by HVboom** [sferik/rails_admin#2142]
The current implementation does not take custom configuration settings for the Ckeditor into account.

Environment:

``` bash
rails -v
Rails 4.1.7

ruby -v
ruby 2.1.5p273 (2014-11-13 revision 48405) [x86_64-darwin14.0]

grep rails_admin Gemfile.lock 
    rails_admin (0.6.6)
  rails_admin
```

Line 166 in file https://github.com/sferik/rails_admin/blob/master/app/assets/javascripts/rails_admin/ra.widgets.coffee has to be changed to use the configuration options only.
- current implementation:

``` coffee
window.CKEDITOR.replace(this, $(this).data('options'))
```
- proposed fix:

``` coffee
window.CKEDITOR.replace(this, $(this).data('options')['options'])
```

With this fix you can setup individual Ckeditor features for different fields, like:

``` ruby
...
  config.model 'TextContent' do
    edit do
      field :name
      field :text, :ck_editor
    end

    show do
      field :name
      field :text do
        pretty_value do
          value.html_safe
        end
      end
    end
  end

  config.model 'Page' do
    edit do
      field :title
      field :keywords
      field :description, :ck_editor do
        # config_js "#{Rails.application.config.assets.prefix}/ckeditor/page.js"
        config_js base_location + "page.js"
      end
    end

    show do
      field :title
      field :keywords
      field :description do
        pretty_value do
          value.html_safe
        end
      end
    end
  end
...
```

Example file `app/assets/javascripts/ckeditor/page.js`:

``` javascript
CKEDITOR.editorConfig = function(config) {
  config.uiColor = "#145900";
  config.toolbarGroups = [
    {
      name: "document",
      groups: ["mode"]
    }, {
      name: "clipboard",
      groups: ["clipboard", "undo"]
    }, {
      name: "basicstyles",
      groups: ["basicstyles", "cleanup"]
    }
  ];
};
```
- Branch with fix: https://github.com/HVboom/rails_admin
- Demo application: https://github.com/HVboom/TestApp/tree/CKEditor

**Comment by ouelletv** [comment]
Bump, this should be on master

**Comment by mshibuya** [comment]
Fixed by c38b76d707f198be0ac8baa1ff02dde7bd02344f.
